<!DOCTYPE html>
<!--[if IE 8 ]><html class="no-js oldie ie8" lang="en"> <![endif]-->
<!--[if IE 9 ]><html class="no-js oldie ie9" lang="en"> <![endif]-->
<!--[if (gte IE 9)|!(IE)]><!--><html class="no-js" lang="en"> <!--<![endif]-->
<head>

   <!--- basic page needs
   ================================================== -->
   <meta charset="utf-8">
	<title>Marcin Magnus, PhD</title>
	<meta name="description" content="">
	<meta name="author" content="">

   <!-- mobile specific metas
   ================================================== -->
	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

 	<!-- CSS
   ================================================== -->
   <link rel="stylesheet" href="css/base.css">
   <link rel="stylesheet" href="css/main.css">
   <link rel="stylesheet" href="css/vendor.css">

   <!-- script
   ================================================== -->
	<script src="js/modernizr.js"></script>

   <!-- favicons
	================================================== -->
	<link rel="icon" type="image/png" href="images/rna.png">

</head>

<body id="top">

	<!-- header
   ================================================== -->
   <header>

   	<div class="row">
   	  <!--<div class="logo">
	         <a href="index.html">Khronos</a>
	      </div>-->

	  <!--
			<div class="social-links">
			   <ul>
					<li><a href="#"><i class="fa fa-facebook"></i></a></li>
					<li><a href="#"><i class="fa fa-twitter"></i></a></li>
				   <li><a href="#"><i class="fa fa-pinterest"></i></a></li>
				   <li><a href="#"><i class="fa fa-instagram"></i></a></li>
				   <li><a href="#"><i class="fa fa-dribbble"></i></a></li>
				</ul>
			</div>
			-->
   	</div>

   </header> <!-- /header -->

   <!-- home
   ================================================== -->
   <section id="home" class="home-particles">

   	<div class="shadow-overlay"></div>

   	<div class="content-wrap-table">

		   <div class="main-content-tablecell">

		   	<div class="row">
		   		<div class="col-twelve">

			  	  <div class="bottom-text">
			  	    <h1>RNAs are like these dots.<br />
				      They <span style="color:orange">evolve</span> from each other.
				      <br />
				      <span style="color:red">But how?</span>
				    </h1>
				    <p style="color:white">
				      Marcin Magnus, PhD</br>
				      <b>Computational RNA biologist at Harvard University, USA</br></b>
				      <!--Trying to understand evolutionary relationship between RNA sequence, structure and function</br>-->
				</br>
I'm trying to understand the evolution and role of RNA and RNPs in biological systems using computational methods (machine learning, molecular dynamics, structure prediction) and experimental (yeast genetics, molecular biology) and codify this knowledge into predictive computational models that can be applied for future RNA therapeutics.
				    </p>

					<a href="http://genesilico.pl/mqapRNA/">mqapRNA</a> | <a href="https://github.com/mmagnus/EvoClustRNA">EvoClustRNA</a> |
					<a href="http://genesilico.pl/RNArchitecture">RNArchitecture</a> |
					<a href="http://genesilico.pl/SimRNAweb">SimRNAweb</a> |
				        <a href="https://github.com/mmagnus/rna-tools">rna-tools</a> | <a href="https://github.com/RNA-Puzzles">RNA Puzzles</a></br>
			    </br>
			<!--	    <a href="https://academic.oup.com/nar/article/45/1/i/2871059" alt="Cover image by Marcin Magnus and Janusz M. Bujnicki (IIMCB,Warsaw): An accurate 3D model of HCV IRES RNA structure, obtained with the fully automated RNA modeling method SimRNAweb, using only RNA sequence as an input. The model has RMSD of 5.52 Å to the experimentally determined structure (PDB id: 1kh6). The superposition has been done and visualized using PyMOL, and the image has been repainted according to the style of “Transverse Line” painting by Wassily Kandinsky (1923), using a machine learning method DeepArt.io. M.M. and J.M.B. thank Siqi Tian (Rhiju Das lab) for inspiration.">Science & Art</a> | -->
			   <a style="" href="https://www.facebook.com/RNAClubWarsaw">RNA Club Warsaw</a> | <a href="http://doscience.iimcb.gov.pl/">Do Science!<br /> </br>
                                        <!-- <a href="https://mmagnus.github.io/notes/blog/">My Notes</a>                                            </br>-->
                                    </br>
			   	    <a href="https://scholar.google.pl/citations?user=O2gLO2EAAAAJ&hl=en">[GoogleScholar]</a>
			   	    <a href="https://publons.com/researcher/1682846/marcin-magnus/">[Publons]</a>
			            <a href="https://github.com/mmagnus">[GitHub]</a>

				    </p>

			   	  </div> <!-- /twelve -->

			   	  <div class="scroll-icon">

				      <p class="scroll-text">Scroll For More Info</p>
				      <a href="#info" class="smoothscroll">
					  <div class="mouse"></div>
				      </a>
				      <div class="end-top"></div>

				  </div> <!-- /scroll-icon -->
		   	        </div> <!-- /row -->

		        </div> <!-- /main-content -->

		   </div> <!-- /content-wrap -->

   </section> <!-- /home -->

   <!-- info
        ================================================== -->
   <section id="info">

       <div class="info-overlay"></div>

       <div class="row">
   	   <div class="col-twelve tabs-wrap">

               <!--
   		    <ul class="tabs">

   		    <li class="active" data-id="tab-about"><i class="icon-user"></i><span>About</span></li>
   		    <li data-id="tab-contact"><i class="icon-mail"></i><span>Contact</span></li>
		    <li data-id="tab-subscribe"><i class="icon-newspaper"></i><span>Subscribe</span></li>
		    </ul>
               -->
	       <div class="tab-container">

		   <!-- tab content - about
   			================================================== -->
		   <div id="tab-about" class="tab-content">

		       <div class="tab-entry">

			   <div class="row tab-entry-intro">
			       <div class="col-twelve with-bottom-line">

				   <h1>Get To Know My Science</h1>

				   <p class="lead">A list of projects I've done with wonderful colleagues.</p>

			       </div>
			   </div> <!-- /tab-content-intro -->

			   <div class="row about-content tab-entry-content">

			       <div class="about-list block-1-3 block-s-1-2 block-tab-full">

				   <div class="bgrid item">
                                       <img style="width:60%" src="images/simrnaweb.png">
				       <div class="item-content">

					   <h3 class="h05"><a style="color:black;" href="https://genesilico.pl/SimRNAweb/">SimRNAweb</a></h3>

					   <p>
                                               SimRNA is a method for RNA 3D structure modeling with optional restraints.
                                               <!-- -
                                                    SimRNA uses a coarse-grained representation, relies on the Monte Carlo method for sampling the conformational space, and employs a statistical potent
                                                    ial to approximate the energy and identify conformations that correspond to biologically relevant structures.-->
					   </p>

				       </div>

				   </div> <!-- /bgrid -->
				   <div class="bgrid item">
                                       <img style="width:66%" src="images/rnarchitecture.png">
				       <div class="item-content">
					   <h3 class="h05"><a style="color:black;" href="https://genesilico.pl/RNArchitecture">RNArchitecture</a></h3>							        <p>RNArchitecture is a structured ncRNAs, with focus on sequence and structure similarities.</p>
				       </div>
				   </div> <!-- /bgrid -->

				   <div class="bgrid item">

                                       <img style="width:90%" src="images/npdock.png">
				       <div class="item-content">

					   <h3 class="h05"><a style="color:black;" href="http://genesilico.pl/NPDock">NPDock</a></h3>

					   <p>NPDock is a web server for modeling of RNA-protein and DNA-protein complex structures.</p>


				       </div>

				   </div> <!-- /bgrid -->

				   <div class="bgrid item">
                                       <img style="width:73%" src="images/evoclust.png">
				       <div class="item-content">
					   <h3 class="h05"><a style="color:black;" href="https://github.com/mmagnus/EvoClustRNA">EvoClustRNA</a></h3>
					   <p>Clustering routines of evolutionary conserved regions (helical regions) for RNA fold prediction.</p>
				       </div>

				   </div> <!-- /bgrid -->

				   <div class="bgrid item">

                                       <img style="width:80%" src="images/cwc15.png">
				       <div class="item-content">
					   <h3 class="h05">Splicing modulation</h3>

					   <p>Cwc15 stabilizes the spliceosomal catalytic interactions in the first-step conformation.</p>
				       </div>

				   </div> <!-- /bgrid -->

				   <div class="bgrid item">

                                       <img style="width:44%" src="images/u6.png">

				       <div class="item-content">
					   <h3 class="h05"><a href="https://github.com/mmagnus/rna-tools/tree/master/U6MolCell" style="color:black">U6 RNA Dynamics</a></h3>

					   <p>The lower U6-ISL stem is flexible and changes during spliceosomal catalysis.
                                               This feature is conserved across eukaryotes, in U6atac and domain V of group II introns.</p>

				       </div>

				   </div> <!-- /bgrid -->

				   <div class="bgrid item">
                                       <img style="width:94%" src="images/rt.png">
				       <div class="item-content">
					   <h3 class="h05"><a href="https://github.com/mmagnus/rna-tools" style="color:black"">rna-tools</a></h3>
					   <p>A toolbox to analyze sequences, structures and simulations of RNA</p>
				       </div>

				   </div> <!-- /bgrid -->

				   <div class="bgrid item">
                                       <img style="width:81%" src="images/rp.jpg">
				       <div class="item-content">
					   <h3 class="h05"><a href="http://www.rnapuzzles.org" style="color:black"">RNA-Puzzles</a></h3>
					   <p>A collective, CASP-like, experiment for blind RNA tertiary structure prediction.</p>
				       </div>

				   </div> <!-- /bgrid -->


				   <div class="bgrid item">
                                       <img style="width:100%;padding-top:4%;" src="images/mqaprna.png">
				       <div class="item-content">
					   <h3 class="h05"><a href="http://genesilico.pl/mqapRNA/" style="color:black"">mqapRNA</a></h3>
					   <p>A quality prediction method for scoring RNA 3D models based on the deep learning.</p>
				       </div>

				   </div> <!-- /bgrid -->




			       </div> <!-- /about-list -->

			   </div> <!-- /row about-content -->

		       </div> <!-- /tab-entry -->

		   </div> <!-- /tab-about -->


	       </div> <!-- /tab-subscribe -->

	       <div class="about-list block-1-2 block-s-1-2 block-tab-full">

		   <div class="bgrid item">
                       <img style="width:30%;" src="images/doscience.png">
                       <div class="item-content">
			   <h3 class="h05"><a href="https://www.facebook.com/DoScience/" style="color:black"">Do Science!</a></h3>
			   <p>Science Club. A place to discuss SCIENCE.</p>
		       </div>
                   </div>

		   <div class="bgrid item">
                       <img style="width:29%;" src="images/rnaclub.png">
                       <div class="item-content">
			   <h3 class="h05"><a href="https://www.facebook.com/RNAClubWarsaw/" style="color:black"">RNA Club Warsaw!</a></h3>					    <p>Do Science! Spin-off. A place to discuss <b>RNA</b> SCIENCE.</p>
		       </div>
                   </div>
               </div>


	       <div class="row tab-entry-intro">
		   <div class="col-twelve with-bottom-line">
		       <h1>My Hacking Tools</h1>
		   </div>
	       </div> <!-- /tab-content-intro -->
	       <div class="about-list block-1-2 block-s-1-2 block-tab-full">
		   <div class="bgrid item">
                       <img style="width:20%;" src="images/emacs.png">
                       <div class="item-content">
			   <h3 class="h05"><a href="https://github.com/mmagnus/emacs-env" style="color:black"">emacs-env</a></h3>
			   <p>Emacs is my rifle. There are many editors like it, but this one is mine.
                               Without me, my rifle is useless. Without my rifle, I am useless. Check out some of the extensions:
			       <a href="https://github.com/mmagnus/emacs-grammarly" style="color:black"">emacs-grammarly</a> 
			       <a href="https://github.com/mmagnus/emacs-pdb-mode" style="color:black"">emacs-pdb-mode</a>
			       <a href="https://github.com/mmagnus/OrgModeClocking2Calendar" style="color:black"">OrgModeClocking2Calendar</a>
			   <a href="https://github.com/mmagnus/garmin2orgmode" style="color:black"">garmin2orgmode</a> </p>			   
		       </div>
                   </div>

		   <div class="bgrid item">
                       <img style="width:60%;padding-top:6%;" src="images/geekbook.png">
                       <div class="item-content">
			   <h3 class="h05"><a href="https://github.com/mmagnus/geekbook" style="color:black">Geekbook</a></h3>
			   <p>Fun to read & fun to write - eXtreme eXtendable note taking system for nerds/geeks (including scientists!) = beautiful html generator of your markdown-based notes.</p>
		       </div>
                   </div>
               </div>


	       <div class="about-list block-1-2 block-s-1-2 block-tab-full">
		   <div class="bgrid item">
                       <img style="width:100%;" src="images/pocketx.png">
		       <h3 class="h05"><a href="https://github.com/mmagnus/Pocket-Plus-Calibre-Plugin" style="color:black">Pocket-Plus-Calibre-Plugin</a></h3>
		       <p>Calibre plugin for Pocket. Now, you get your articles organized by your Pocket tags, and way more...</p>
		   </div>

		   <div class="bgrid item">
                       <div class="item-content">
		       <h3 class="h05">
			 <a href="https://github.com/mmagnus/pubmex" style="color:black">pubmex</a>
		       </h3>
		     <p>is a script to get a fancy/full paper title based on analysis of the file (can be also combined with macOS Finder)</p>
		     <pre style="font-size:80%; padding:2%;line-height:120%">
$ pubmex.py sharp2017.pdf
Sharp.Hockfield.Convergence.The.future.health.Science.2017.pdf</pre>
		     </div>
		     </center>
                   </div>
               </div>



               <h1 class="">Papers</h1>
               <div style="text-align:center;background:#F1F1F1;padding:2%;">
                   <a href="https://scholar.google.com/citations?user=O2gLO2EAAAAJ&hl=en">Google Scholar</a>: Citations 710, H-index: 12; Publons: Peer-review: 12 [as of 210428].
                                </div>
                                   </br>
	<div style="text-align:left;background:#F1F1F1;padding:3%;word-wrap: normal;word-break:initial;hyphens: none;">
		<h2>Selected papers</h2>
@ Stanford University, CA, USA & International Institute of Molecular and Cell Biology, Poland 
<p><b>RNA 3D Structure Prediction Using Multiple Sequence Alignment Information.</b><br>
    <u>M.&nbsp;Magnus</u>, K.&nbsp;Kappel, R.&nbsp;Das, and J.&nbsp;M.&nbsp;Bujnicki<br>
    <i>BMC Bioinformatics, vol. 20, no. 1, pp. 512–15, Oct. 2019</i><br>
    <a href="https://github.com/mmagnus/EvoClustRNA">https://github.com/mmagnus/EvoClustRNA</a></p>

<hr>		
@ Warsaw University, Poland
		<p><b>Rearrangements within the U6 snRNA core at the transition between the two catalytic steps of splicing.</b><br>
    K.&nbsp;Eysmont, K.&nbsp;Matylla-Kulinska, A.&nbsp;Jaskulska, <u>M.&nbsp;Magnus</u>, and M.&nbsp;M.&nbsp;Konarska<br> <i>Molecular Cell, vol. 75, no. 3, pp. 538–548.e3, Aug. 2019</i></b></u></br>
    <a href="https://github.com/mmagnus/rna-tools/tree/master/U6MolCell">https://github.com/mmagnus/rna-tools/tree/master/U6MolCell</a></p>

<hr>

@ International Institute of Molecular and Cell Biology, Poland & Warsaw University, Poland
<p><b>RNA-Puzzles toolkit: A&nbsp;computational resource of RNA 3D structure benchmark datasets,
    structure manipulation, and evaluation tools.</b><br>
    <u>M.&nbsp;Magnus</u>, M.&nbsp;Antczak, P.&nbsp;Lukasiak, J.&nbsp;Wiedemann, T.&nbsp;Zok, J.&nbsp;M.&nbsp;Bujnicki, E.&nbsp;Westhof, M.&nbsp;Szachniuk, and Z.&nbsp;Miao<br>
    <i>Nucleic Acid Research, vol. 48, no. 2, pp. 576–588, Jan. 2020</i><br>
    <a href="https://github.com/mmagnus/rna-tools">https://github.com/mmagnus/rna-tools</a></p>

@ International Institute of Molecular Mechanisms and Machines, Polish Academy of Sciences, Warsaw, Poland
<p><b> rna-tools.online – a Swiss army knife for RNA 3D modeling workflows</b><br>
    <u>M. Magnus</u></br>
    <i>Nucleic Acid Research, 10.1093/nar/gkac372, May 2022</i></br>
    <a href="https://rna-tools.online">https://rna-tools.online</a></p>

				   </div>
		       </br>
<div style="text-align:left;background:#F1F1F1;padding:3%;word-wrap: normal;word-break:initial;hyphens: none;">

<h4>Published:</h4>

<p>[22]<b> RNA Puzzles V @ <i>Nature Methods</i></b> 2024</br>
	
<p>[21]<b> SimRNAweb v2.0: A web server for RNA folding simulations and 3D structure modeling, with optional restraints and enhanced analysis of folding trajectories @ <i>Nucleic Acid Research</i></b> 2024<br>
	
<p>[20]<b> RNA3DB: A dataset for training and benchmarking deep learning models for RNA structure prediction</b><br>
M. Szikszai, <u>M. Magnus</u>, S. Sanghi, S. Kadyan, N. Bouatta, E. Rivas<br>
<a href="https://www.biorxiv.org/content/10.1101/2024.01.30.578025v2">@biorxiv</a> <i>2024</i> <br>
</p>
	

				   <p>[19]<b> RNA fragment assembly with experimental restraints.</b><br>
				   G. Chojnowski, R. Zaborowski, <u>M. Magnus</u>, S. Mukherjee, and J. M. Bujnicki<br>
				   <a href="https://www.biorxiv.org/content/10.1101/2021.02.08.430198v1.full.pdf+html">@biorxiv</a> <i>2021</i> 
	 <a href="https://academic.oup.com/bioinformatics/article/39/9/btad527/7255907">Bioinformatics, Volume 39, Issue 9, September 2023</a></br>
				   <a href="http://iimcb.genesilico.pl/rnamasonry">http://iimcb.genesilico.pl/rnamasonry</a></br>
	
<p>[18] <b>RNA 3D structure comparison using RNA-Puzzles toolkit.</b><br>
  <u>M.&nbsp;Magnus</u>, Z.&nbsp;Miao<br>
  <i>RNA Structure Prediction, 263-285, Jan 2023</i></br>
  <a href="https://github.com/mmagnus/rna-tools">https://github.com/mmagnus/rna-tools</a></p>		
<p>[17]<b> rna-tools.online – a Swiss army knife for RNA 3D modeling workflows.</b><br>
    <u>M. Magnus</u></br>
    <i>Nucleic Acid Research, 10.1093/nar/gkac372, May 2022</i></br>
    <a href="https://rna-tools.online">https://rna-tools.online</a></p>

<p>[16]<b> RNA-Puzzles Round IV: 3D structure predictions of four ribozymes and two aptamers.</b><br>
    Z. Miao, R. W. Adamiak, M. Antczak, M. J. Boniecki, J. M. Bujnicki, S.-J. Chen, C. Y. Cheng, 
    Y. Cheng, F.-C. Chou, R. Das, N. V. Dokholyan, F. Ding, C. Geniesse, Y. Jiang, A. Joshi, A. Krokhotin, 
    <u>M. Magnus</u>, 
    O. Mailhot, F. Major, T. H. Mann, P. Piatkowski, R. Pluta, 
    M. Popenda, J. Sarzynska, L. Sun, M. Szachniuk, S. Tian, J. Wang, 
    J. Wang, A. M. Watkins, J. Wiedemann, Y. Xiao, X. Xu, J. D. Yesselman, 
    D. Zhang, Y. Zhang, Z. Zhang, C. Zhao, P. Zhao, Y. Zhou, T. Zok, 
    A. Zyła, A. Ren, R. T. Batey, B. L. Golden, L. Huang, D. M. Lilley, 
    Y. Liu, D. J. Patel, and E. Westhof</br>
    <i>RNA, [epub] 10.1261/rna.075341.120, May 2020</i></br>

<p>[15]<b> RNA-Puzzles toolkit: A&nbsp;computational resource of RNA 3D structure benchmark datasets,
    structure manipulation, and evaluation tools.</b><br>
    <u>M.&nbsp;Magnus</u>, M.&nbsp;Antczak, P.&nbsp;Lukasiak, J.&nbsp;Wiedemann, T.&nbsp;Zok, J.&nbsp;M.&nbsp;Bujnicki, E.&nbsp;Westhof, M.&nbsp;Szachniuk, and Z.&nbsp;Miao<br>
    <i>Nucleic Acid Research, vol. 48, no. 2, pp. 576–588, Jan. 2020</i><br>
    <a href="https://github.com/mmagnus/rna-tools">https://github.com/mmagnus/rna-tools</a></p>

<p>[14]<b> RNA 3D Structure Prediction Using Multiple Sequence Alignment Information.</b><br>
    <u>M.&nbsp;Magnus</u>, K.&nbsp;Kappel, R.&nbsp;Das, and J.&nbsp;M.&nbsp;Bujnicki<br>
    <i>BMC Bioinformatics, vol. 20, no. 1, pp. 512–15, Oct. 2019</i><br>
    <a href="https://github.com/mmagnus/EvoClustRNA">https://github.com/mmagnus/EvoClustRNA</a></p>

<p style="background-color:lightgray;padding:1%;">[13] <b>Rearrangements within the U6 snRNA core at the transition between the two catalytic steps of splicing.</b><br>
    K.&nbsp;Eysmont, K.&nbsp;Matylla-Kulinska, A.&nbsp;Jaskulska, <u>M.&nbsp;Magnus</u>, and M.&nbsp;M.&nbsp;Konarska<br> <i>Molecular Cell, vol. 75, no. 3, pp. 538–548.e3, Aug. 2019</i></b></u></br>
    <a href="https://github.com/mmagnus/rna-tools/tree/master/U6MolCell">https://github.com/mmagnus/rna-tools/tree/master/U6MolCell</a></p>



<p>[12]<b> RNArchitecture: a&nbsp;database and a&nbsp;classification system of RNA families, with a&nbsp;focus on structural information.</b><br>
    P.&nbsp;Boccaletto, <u>M.&nbsp;Magnus</u>, C.&nbsp;Almeida, A.&nbsp;Zyła, A.&nbsp;Astha, R.&nbsp;Pluta, B.&nbsp;Bagiński, E.&nbsp;J.&nbsp;Jankowska, S.&nbsp;Dunin-Horkawicz, T.&nbsp;K.&nbsp;Wirecki, M.&nbsp;J.&nbsp;Boniecki, F.&nbsp;Stefaniak, and J.&nbsp;M.&nbsp;Bujnicki<br>
    <i>Nucleic Acids Research, vol. 46, no. 1, pp. D202–D205, Jan. 2018</i><br>
<a href="http://genesilico.pl/RNArchitecture">http://genesilico.pl/RNArchitecture</a></p>

<p>[11]<b> RNA-Puzzles Round III: 3D RNA structure prediction of five riboswitches and one ribozyme.</b></br>
Z.&nbsp;Miao, R.&nbsp;W.&nbsp;Adamiak, M.&nbsp;Antczak, R.&nbsp;T.&nbsp;Batey, A.&nbsp;J.&nbsp;Becka, M.&nbsp;Biesiada, M.&nbsp;J.&nbsp;Boniecki, J.&nbsp;M.&nbsp;Bujnicki, S.-J.&nbsp;Chen, C.&nbsp;Y.&nbsp;Cheng, F.-C.&nbsp;Chou, A.&nbsp;R.&nbsp;Ferré-D'Amaré, R.&nbsp;Das, W.&nbsp;K.&nbsp;Dawson, F.&nbsp;Ding, N.&nbsp;V.&nbsp;Dokholyan, S.&nbsp;Dunin-Horkawicz, C.&nbsp;G
eniesse, K.&nbsp;Kappel, W.&nbsp;Kladwang, A.&nbsp;Krokhotin, G.&nbsp;E.&nbsp;Łach,&nbsp;F.&nbsp;Major, T.&nbsp;H.&nbsp;Mann, <u>M.&nbsp;Magnus</u>, K.&nbsp;Pachulska-Wieczorek, D.&nbsp;J.&nbsp;Patel, J.&nbsp;A.&nbsp;Piccirilli, M.&nbsp;Popenda, K.&nbsp;J.&nbsp;Purzycka, A.&nbsp;Ren, G.&nbsp;M.&nbsp;Rice, J.&nbsp;Santalucia, J.&nbsp;Sarzynska, M.&nbsp;Szachniuk, A.&nbsp;Tandon, J.&nbsp;J.&nbsp;Trausch, S.&nbsp;Tian, J.&nbsp;Wang, K.&nbsp;M.&nbsp;Weeks, B.&nbsp;Williams, Y.&nbsp;Xiao, X.&nbsp;Xu, D.&nbsp;Zhang, T.&nbsp;Zok, and E.&nbsp;Westhof<br>
<i>RNA, vol.&nbsp;23, no.&nbsp;5, pp.&nbsp;655–672, May&nbsp;2017</i></p>

<p>[10]<b> SimRNAweb: a&nbsp;web server for RNA 3D structure modeling with optional restraints.</b><br>
<u>M.&nbsp;Magnus* </u>, M.&nbsp;J.&nbsp;Boniecki*, W.&nbsp;K.&nbsp;Dawson, and J.&nbsp;M.&nbsp;Bujnicki<br>
<i>Nucleic Acids Research, vol.&nbsp;44, no.&nbsp;1, pp.&nbsp;W315–9, Jul.&nbsp;2016</i><br><a hfef="https://genesilico.pl/SimRNAweb"/>https://genesilico.pl/SimRNAweb</a></p>

<p>[9]<b> Modeling of Protein-RNA Complex Structures Using Computational Docking Methods.</b><br>
B.&nbsp;Madan, J.&nbsp;M.&nbsp;Kasprzak, I.&nbsp;Tuszyńska, <u>M.&nbsp;Magnus</u>, K.&nbsp;Szczepaniak, W.&nbsp;K.&nbsp;Dawson, and J.&nbsp;M.&nbsp;Bujnicki<br>
<i>Methods Mol.&nbsp;Biol., vol.&nbsp;1414, no.&nbsp;21, pp.&nbsp;353–372,&nbsp;2016</i></p>

<p>[8]<b> RNA 3D Structure Modeling by Combination of Template-Based Method ModeRNA, Template-Free Folding with SimRNA, and Refinement with QRNAS.</b><br> P.&nbsp;Piatkowski, J.&nbsp;M.&nbsp;Kasprzak, D.&nbsp;Kumar, <u>M.&nbsp;Magnus</u>, G.&nbsp;Chojnowski, and J.&nbsp;M.&nbsp;Bujnicki<br>
<i>Methods Mol.&nbsp;Biol., vol.&nbsp;1490, no.&nbsp;Suppl, pp.&nbsp;217–235,&nbsp;2016</i></p>

<p>[7]<b> NPDock: a&nbsp;web server for protein-nucleic acid docking.</b><br>
I.&nbsp;Tuszyńska, <u>M.&nbsp;Magnus</u>, K.&nbsp;Jonak, W.&nbsp;K.&nbsp;Dawson, and J.&nbsp;M.&nbsp;Bujnicki<br>
<i>Nucleic Acids Research, vol.&nbsp;43, no.&nbsp;1, pp.&nbsp;W425–30, Jul.&nbsp;2015</i><br>
<a href="http://genesilico.pl/NPDock">http://genesilico.pl/NPDock</a></p>

<p>[6]<b> RNA-Puzzles Round II: assessment of RNA structure prediction programs applied to three large RNA structures.</b><br>
Z.&nbsp;Miao, R.&nbsp;W.&nbsp;Adamiak, M.-F.&nbsp;Blanchet, M.&nbsp;J.&nbsp;Boniecki, J.&nbsp;M.&nbsp;Bujnicki, S.-J.&nbsp;Chen, C.&nbsp;Y.&nbsp;Cheng, G.&nbsp;Chojnowski, F.-C.&nbsp;Chou, P.&nbsp;Cordero, J.&nbsp;A.&nbsp;Cruz, A.&nbsp;R.&nbsp;Ferré-D'Amaré, R.&nbsp;Das, F.&nbsp;Ding, N.&nbsp;V.&nbsp;Dokholyan, S.&nbsp;Dunin-Horkawicz, W.&nbsp;Kladwang, A.&nbsp;Krokhotin, G.&nbsp;Lach, <u>M.&nbsp;Magnus</u>, F.&nbsp;Major, T.&nbsp;H.&nbsp;Mann, B.&nbsp;Masquida, D.&nbsp;Matelska, M.&nbsp;Meyer, A.&nbsp;Peselis, M.&nbsp;Popenda, K.&nbsp;J.&nbsp;Purzycka, A.&nbsp;Serganov, J.&nbsp;Stasiewicz, M.&nbsp;Szachniuk, A.&nbsp;Tandon, S.&nbsp;Tian, J.&nbsp;Wang, Y.&nbsp;Xiao, X.&nbsp;Xu, J.&nbsp;Zhang, P.&nbsp;Zhao, T.&nbsp;Zok, and E.&nbsp;Westhof<br>
<i>RNA, vol.&nbsp;21, no.&nbsp;6, pp.&nbsp;1066–1084, Jun.&nbsp;2015</i></p>

<p>[5]<b> Computational modeling of protein-RNA complex structures.</b><br>
I.&nbsp;Tuszyńska, D.&nbsp;Matelska, <u>M.&nbsp;Magnus</u>, G.&nbsp;Chojnowski, J.&nbsp;M.&nbsp;Kasprzak, L.&nbsp;P.&nbsp;Kozlowski, S.&nbsp;Dunin-Horkawicz, and J.&nbsp;M.&nbsp;Bujnicki<br>
<i>Methods, vol.&nbsp;65, no.&nbsp;3, pp.&nbsp;310–319, Feb.&nbsp;2014</i></p>

<p>[4]<b> Computational modeling of RNA 3D structures, with the aid of experimental restraints.</b><br>
<u>M.&nbsp;Magnus*</u>, D.&nbsp;Matelska*, G.&nbsp;Lach, G.&nbsp;Chojnowski, M.&nbsp;J.&nbsp;Boniecki, E.&nbsp;Purta, W.&nbsp;K.&nbsp;Dawson, S.&nbsp;Dunin-Horkawicz, and J.&nbsp;M.&nbsp;Bujnicki<br>
<i>RNA Biol, vol.&nbsp;11, no.&nbsp;5, pp.&nbsp;522–536,&nbsp;2014</i></p>

<p>[3]<b> MetaLocGramN: A meta-predictor of protein subcellular localization for Gram-negative bacteria.</b><br>
<u>M.&nbsp;Magnus</u>, M.&nbsp;Pawlowski, and J.&nbsp;M.&nbsp;Bujnicki<br>
<i>BBA&nbsp;-&nbsp;Proteins and Proteomics, vol.&nbsp;1824, no.&nbsp;12, pp.&nbsp;1425–1433, Dec.&nbsp;2012</i><br> <a href="http://genesilico.pl/MetaLocGramN/">http://genesilico.pl/MetaLocGramN/</a></p>

<p>[2]<b> Structural bioinformatics of the human spliceosomal proteome.</b><br>
I. Korneta, <u>M.&nbsp;Magnus</u>, and&nbsp;J.&nbsp;M.&nbsp;Bujnicki<br>
<i>Nucleic Acids Research, vol.&nbsp;40, no.&nbsp;15, pp.&nbsp;7046–7065,&nbsp;Aug.&nbsp;2012</i><br> <a href="http://iimcb.genesilico.pl/SpliProt3D">http://iimcb.genesilico.pl/SpliProt3D</a></p>

<p>[1]<b> Structures of MicroRNA Precursors.</b><br>
P. Kozlowski, J.&nbsp;Starega-Roslan, M.&nbsp;Legacz, <u>M.&nbsp;Magnus</u>, and W.&nbsp;J.&nbsp;Krzyzosiak<br>
<i>Current Perspectives in microRNAs (miRNA), no.&nbsp;1,&nbsp;Dordrecht: Springer, Dordrecht, 2008, pp.&nbsp;1–16.</i></p>

</div>
</pre>
<br>
<h1 class="">Psychedelic-assisted Psychotherapy</h1>
<p>Zendo SIT trained psychodelic sitter</p>
		 <img style="width:40%" src="images/sit1.jpg">
    
            <div> <!-- /tab-container -->
 <style>
        ul {
            list-style-position: inside;
            text-align: center;
       }
  </style>
  
<p>Courses:
  <ul>
    <li>Zendo SIT - Sitting and Integration Training Psychedelic Care and Harm Reduction Live Course, USA (December 7-8, 2024)</li>
    <li>MAPS - Psychedelic Fundamentals Course, USA (December, 2024)</li>
  </ul>

Conferences:
<ul>
  <li>4th Annual Psychedelic Therapeutics & Drug Development, Boston, USA (May 23-24, 2024)</li>
  <li>Psychodelic Science (Nauka Psychodeliczna) 2024, University of Warsaw, Warsaw, Poland (June 15-16, 2024)</li>
<li>4th Annual Conference on Psychedelics and Psychedelic Medicine Boston, Harvard Psychiatry, Boston, USA (November 15-16, 2024)</li>
  <li>Psychedelic Science Conference 2025, Colorado, USA (June 16-20, *2025*)</li>
</ul>
</p>
   		</div> <!-- twelve -->
   	</div> <!-- row -->


   </section> <!-- /info -->




   <!-- footer
   ================================================== -->
   <footer>

   	</div> <!-- /footer-bottom -->

   	<div class="back-to-top">
		 	<a href="#top" class="smoothscroll"><span>Back to Top</span></a>
		</div>

   </footer>

   <div id="preloader">
    	<div id="loader"></div>
   </div>

   <!-- Java Script
   ================================================== -->
   <script src="js/jquery-2.1.3.min.js"></script>
   <script src="js/plugins.js"></script>
   <script src="js/main.js"></script>

</body>

</html>
