<!DOCTYPE html>
<!--[if IE 8 ]><html class="no-js oldie ie8" lang="en"> <![endif]-->
<!--[if IE 9 ]><html class="no-js oldie ie9" lang="en"> <![endif]-->
<!--[if (gte IE 9)|!(IE)]><!--><html class="no-js" lang="en"> <!--<![endif]-->
<head>

   <!--- basic page needs
   ================================================== -->
   <meta charset="utf-8">
	<title>Marcin Magnus, PhD</title>
	<meta name="description" content="">  
	<meta name="author" content="">

   <!-- mobile specific metas
   ================================================== -->
	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

 	<!-- CSS
   ================================================== -->
   <link rel="stylesheet" href="css/base.css">  
   <link rel="stylesheet" href="css/main.css">
   <link rel="stylesheet" href="css/vendor.css">

   <!-- script
   ================================================== -->
	<script src="js/modernizr.js"></script>

   <!-- favicons
	================================================== -->
	<link rel="icon" type="image/png" href="images/rna.png">

</head>

<body id="top">

	<!-- header 
   ================================================== -->
   <header>

   	<div class="row">
   	  <!--<div class="logo">
	         <a href="index.html">Khronos</a>
	      </div>-->

	  <!--
			<div class="social-links">
			   <ul>
					<li><a href="#"><i class="fa fa-facebook"></i></a></li>
					<li><a href="#"><i class="fa fa-twitter"></i></a></li>
				   <li><a href="#"><i class="fa fa-pinterest"></i></a></li>				   
				   <li><a href="#"><i class="fa fa-instagram"></i></a></li> 
				   <li><a href="#"><i class="fa fa-dribbble"></i></a></li>				     			
				</ul>
			</div>
			-->
   	</div> 

   </header> <!-- /header -->   

   <!-- home
   ================================================== -->
   <section id="home" class="home-particles">

   	<div class="shadow-overlay"></div>

   	<div class="content-wrap-table">		   

		   <div class="main-content-tablecell">

		   	<div class="row">
		   		<div class="col-twelve">

			  	  <div class="bottom-text">
			  	    <h1>RNAs are like these dots.<br />
				      They <span style="color:orange">evolve</span> from each other.
				      <br />
				      <span style="color:red">But how?</span>
				    </h1>
				    <p style="color:white">
				      Marcin Magnus, PhD</br>
				      RNA sequence <> structure <> function</br>

					<a href="http://genesilico.pl/mqapRNA/">mqapRNA</a> | RNA DCA/ExpData | <a href="https://github.com/mmagnus/EvoClustRNA">EvoClustRNA</a> |
					<a href="http://genesilico.pl/RNArchitecture">RNArchitecture</a> |
					<a href="http://genesilico.pl/SimRNAweb">SimRNAweb</a> |
				<a href="https://github.com/mmagnus/rna-tools">rna-tools</a> | <a href="https://github.com/RNA-Puzzles">RNA Puzzles</a> | 
					<a href="https://github.com/mmagnus/rna-pdb-tools/blob/master/rp18.ipynb">rna-structure-notebook.io</a> </br>
					<a href="https://cent.uw.edu.pl/en/laboratories/laboratorium-biologii-rna/#!#z2">In a wet lab with yeast genetics with prof. Magda Konarska</a>
					</br>
					  <a href="https://academic.oup.com/nar/article/45/1/i/2871059" alt="Cover image by Marcin Magnus and Janusz M. Bujnicki (IIMCB,Warsaw): An accurate 3D model of HCV IRES RNA structure, obtained with the fully automated RNA modeling method SimRNAweb, using only RNA sequence as an input. The model has RMSD of 5.52 Å to the experimentally determined structure (PDB id: 1kh6). The superposition has been done and visualized using PyMOL, and the image has been repainted according to the style of “Transverse Line” painting by Wassily Kandinsky (1923), using a machine learning method DeepArt.io. M.M. and J.M.B. thank Siqi Tian (Rhiju Das lab) for inspiration.">Science & Art</a> | <a style="" href="https://www.facebook.com/RNAClubWarsaw">RNA Club Warsaw</a> | <a href="http://doscience.iimcb.gov.pl/">Do Science!<br /> </br>
			   		  <a href="https://scholar.google.pl/citations?user=O2gLO2EAAAAJ&hl=en">[GoogleScholar]</a> 
			   	<a href="https://publons.com/researcher/1682846/marcin-magnus/">[Publons]</a>
			   <a href="https://github.com/mmagnus">[GitHub]</a>					  					  

				    </p>			  				

			   	</div> <!-- /twelve --> 

			   	<div class="scroll-icon">

					   <p class="scroll-text">Scroll For More Info</p>
					   <a href="#info" class="smoothscroll">
					   	<div class="mouse"></div>
					   </a>
				   	<div class="end-top"></div>

					</div> <!-- /scroll-icon -->
		   	</div> <!-- /row -->  

		   </div> <!-- /main-content --> 
		   
		</div> <!-- /content-wrap -->
   
   </section> <!-- /home -->

   <!-- info
   ================================================== -->
   <section id="info">

   	<div class="info-overlay"></div>

   	<div class="row">   		
   		<div class="col-twelve tabs-wrap">

                    <!-- 
   			<ul class="tabs"> 

   				<li class="active" data-id="tab-about"><i class="icon-user"></i><span>About</span></li>
   				<li data-id="tab-contact"><i class="icon-mail"></i><span>Contact</span></li>							
					<li data-id="tab-subscribe"><i class="icon-newspaper"></i><span>Subscribe</span></li>				
				</ul> 
                     -->
				<div class="tab-container">

					<!-- tab content - about
   				================================================== -->
					<div id="tab-about" class="tab-content">
						
					   <div class="tab-entry">

					   	<div class="row tab-entry-intro">
			   		            <div class="col-twelve with-bottom-line">

					   			<h1>Get To Know My Science</h1>

					   			<!-- <p class="lead">Investigation of RNA evolution.</p>-->

					   		</div>   		
					   	</div> <!-- /tab-content-intro -->

					   	<div class="row about-content tab-entry-content">
					     		
					   	    <div class="about-list block-1-3 block-s-1-2 block-tab-full">

						      	<div class="bgrid item">	
                                                            <img style="width:60%" src="images/simrnaweb.png">
						                <div class="item-content">	

						            	    <h3 class="h05"><a style="color:black;" href="https://genesilico.pl/SimRNAweb/">SimRNAweb</a></h3>

							            <p>
                                                                        SimRNA is a method for RNA 3D structure modeling with optional restraints. 
                                                                        SimRNA uses a coarse-grained representation, relies on the Monte Carlo method for sampling the conformational space, and employs a statistical potent
                                                                        ial to approximate the energy and identify conformations that correspond to biologically relevant structures.
						         	    </p>
						         	    
						         	</div> 	         	 

							</div> <!-- /bgrid -->
						      	<div class="bgrid item">	
                                                            <img style="width:66%" src="images/rnarchitecture.png">
						            <div class="item-content">	
						            	<h3 class="h05"><a style="color:black;" href="https://genesilico.pl/RNArchitecture">RNArchitecture</a></h3>
							        <p>RNArchitecture is a database that provides a comprehensive description of relationships between known families of structured ncRNAs, with focus on sequence and structure similarities.</p>
						            </div> 	         	 
							</div> <!-- /bgrid -->

							    <div class="bgrid item">	

                                                                <img style="width:90%" src="images/npdock.png">
						                <div class="item-content">
	                                                            
						            	    <h3 class="h05"><a style="color:black;" href="http://genesilico.pl/NPDock">NPDock</a></h3>  

							            <p>NPDock (Nucleic acid-Protein Dock) is a web server for modeling of RNA-protein and DNA-protein complex structures. 
                                                                        It combines GRAMM for global macromolecular docking, scoring with a statistical potential, clustering of best-scored structures, and local refinement.
						         	    </p>

						         	    
						                </div>	                          

							    </div> <!-- /bgrid -->

							    <div class="bgrid item">
                                                                <img style="width:73%" src="images/evoclust.png">
						                <div class="item-content">
						            	    <h3 class="h05"><a style="color:black;" href="https://github.com/mmagnus/EvoClustRNA">EvoClustRNA</a></h3>
							                <p>Clustering routines of evolutionary conserved regions (helical regions) for RNA fold prediction.</p>
						                </div> 	            	               

							    </div> <!-- /bgrid -->

							    <div class="bgrid item">

                                                                <img style="width:80%" src="images/cwc15.png">
						            <div class="item-content">
						            	<h3 class="h05">Splicing modulation</h3>

							        <p>Cwc15 stabilizes the spliceosomal catalytic interactions in the first-step conformation.</p> 
						            </div>                

							    </div> <!-- /bgrid -->

							    <div class="bgrid item">

                                                                <img style="width:44%" src="images/u6.png">

						                <div class="item-content">	
						            	    <h3 class="h05"><a href="https://github.com/mmagnus/rna-tools/tree/master/U6MolCell" style="color:black">U6 RNA Dynamics</a></h3>

							            <p>The lower U6-ISL stem is flexible and changes during spliceosomal catalysis.
                                                                        This feature is conserved across eukaryotes, in U6atac and domain V of group II introns.</p> 

						            </div>	               

								   </div> <!-- /bgrid -->

								   <div class="bgrid item">
                                                                       <img style="width:94%" src="images/rt.png">
						                       <div class="item-content">
						            	           <h3 class="h05"><a href="https://github.com/mmagnus/rna-tools" style="color:black"">rna-tools</a></h3>
							                   <p>A toolbox to analyze sequences, structures and simulations of RNA</p> 
						                       </div>	               

								   </div> <!-- /bgrid -->

								   <div class="bgrid item">
                                                                       <img style="width:81%" src="images/rp.jpg">
						                       <div class="item-content">
						            	           <h3 class="h05"><a href="http://www.rnapuzzles.org" style="color:black"">RNA-Puzzles</a></h3>
							                   <p>A collective, CASP-like, experiment for blind RNA tertiary structure prediction.</p> 
						                       </div>	               

								   </div> <!-- /bgrid -->


								   <div class="bgrid item">
                                                                       <img style="width:100%;padding-top:4%;" src="images/mqaprna.png">
						                       <div class="item-content">
						            	           <h3 class="h05"><a href="http://genesilico.pl/mqapRNA/" style="color:black"">mqapRNA</a></h3>
							                   <p>A quality prediction method for scoring RNA 3D models based on the deep learning.</p> 
						                       </div>	               

								   </div> <!-- /bgrid -->

                                                                   


						    </div> <!-- /about-list --> 

					     	</div> <!-- /row about-content -->
					     	
					   </div> <!-- /tab-entry -->

					</div> <!-- /tab-about -->	


				</div> <!-- /tab-subscribe -->										

				<div class="about-list block-1-2 block-s-1-2 block-tab-full">

				    <div class="bgrid item">	
                                        <img style="width:30%;" src="images/doscience.png">
                                        <div class="item-content">
					    <h3 class="h05"><a href="https://www.facebook.com/DoScience/" style="color:black"">Do Science!</a></h3>
					    <p>Science Club. A place to discuss SCIENCE.</p> 
				        </div>
                                    </div>

				    <div class="bgrid item">	
                                        <img style="width:29%;" src="images/rnaclub.png">
                                        <div class="item-content">
					    <h3 class="h05"><a href="https://www.facebook.com/RNAClubWarsaw/" style="color:black"">RNA Club Warsaw!</a></h3>					    <p>Do Science! Spin-off. A place to discuss <b>RNA</b> SCIENCE.</p> 
				        </div>
                                    </div>
                                </div>


				<div class="row tab-entry-intro">
			   	    <div class="col-twelve with-bottom-line">
					<h1>My Hacking Tools</h1>
				    </div>   		
				</div> <!-- /tab-content-intro -->
				<div class="about-list block-1-2 block-s-1-2 block-tab-full">
				    <div class="bgrid item">	
                                        <img style="width:20%;" src="images/emacs.png">
                                        <div class="item-content">
					    <h3 class="h05"><a href="https://github.com/mmagnus/emacs-env" style="color:black"">Emacs</a></h3>
					    <p>Emacs is my rifle. There are many editors like it, but this one is mine. 
                                                Without me, my rifle is useless. Without my rifle, I am useless.</p> 
				        </div>
                                    </div>

				    <div class="bgrid item">	
                                        <img style="width:60%;padding-top:6%;" src="images/geekbook.png">
                                        <div class="item-content">
					    <h3 class="h05"><a href="https://github.com/mmagnus/geekbook" style="color:black"">Geekbook</a></h3>
					    <p>Fun to read & fun to write - eXtreme eXtendable note taking system for nerds/geeks (including scientists!) = beautiful html generator of your markdown-based notes.</p> 
				        </div>
                                    </div>
                                </div>




                                <h1 class="">Papers</h1>
                                <pre style="text-align:center;">
Google Scholar: Citations 403, H-index: 9; Publons: Peer-review: 12 [as of 190417].</pre>

                                <pre style="text-align:left;">
In revision:

Rearrangements within the U6 snRNA core during the transition between the two catalytic steps of splicing
Eysmont K., Matylla-Kulińska K., Jaskulska A., Magnus M., Konarska M.
Molecular Cell
                                </pre>
                                <pre style="text-align:left;">
In preparation:

mqapRNA - quality assessment of RNA 3D models
M. Magnus, A. Bogdanowicz, and J. M. Bujnicki
http://genesilico.pl/mqapRNA


RNA-Puzzles Bench: A computational resource of RNA 3D structure benchmark datasets, assessment metrics and structure manipulation tools 
Z. Miao*, M. Magnus*, M. Antczak, P. Lukasiak, M. Szachniuk, J. Wiedemann, T. Zok, J. M. Bujnicki, and E. Westhof
https://github.com/mmagnus/rna-tools</pre>
                                <pre style="text-align:left;">
						    
Submitted:
						    
[14] RNA 3D Structure Prediction Using Multiple Sequence Alignment Information
M. Magnus, K. Kappel, R. Das, and J. M. Bujnicki
https://github.com/mmagnus/EvoClustRNA
BMC Bioinformatics, 2018					  
						    
Published:

[13] Eysmont, K., Matylla-Kulinska, K., Jaskulska, A., Magnus, M., & Konarska, M. M.
Rearrangements within the U6 snRNA core at the transition between the two catalytic steps of splicing. 
Molecular Cell (in press) 2018 
						    
[12] RNArchitecture: a database and a classification system of RNA families, with a focus on structural information
P. Boccaletto, M. Magnus, C. Almeida, A. Zyła, A. Astha, R. Pluta, B. Bagiński, E. J. Jankowska, S. Dunin-Horkawicz, T. K. Wirecki, M. J. Boniecki, F. Stefaniak, and J. M. Bujnicki
Nucleic Acids Research, vol. 46, no. 1, pp. D202–D205, Jan. 2018. http://genesilico.pl/RNArchitecture

[11] RNA-Puzzles Round III: 3D RNA structure prediction of five riboswitches and one ribozyme
Z. Miao, R. W. Adamiak, M. Antczak, R. T. Batey, A. J. Becka, M. Biesiada, M. J. Boniecki, J. M. Bujnicki, S.-J. Chen, C. Y. Cheng, F.-C. Chou, A. R. Ferré-D'Amaré, R. Das, W. K. Dawson, F. Ding, N. V. Dokholyan, S. Dunin-Horkawicz, C. Geniesse, K. Kappel, W. Kladwang, A. Krokhotin, G. E. Łach, F. Major, T. H. Mann, M. Magnus, K. Pachulska-Wieczorek, D. J. Patel, J. A. Piccirilli, M. Popenda, K. J. Purzycka, A. Ren, G. M. Rice, J. Santalucia, J. Sarzynska, M. Szachniuk, A. Tandon, J. J. Trausch, S. Tian, J. Wang, K. M. Weeks, B. Williams, Y. Xiao, X. Xu, D. Zhang, T. Zok, and E. Westhof, RNA, vol. 23, no. 5, pp. 655–672, May 2017.

[10] SimRNAweb: a web server for RNA 3D structure modeling with optional restraints
M. Magnus*, M. J. Boniecki*, W. K. Dawson, and J. M. Bujnicki
Nucleic Acids Research, vol. 44, no. 1, pp. W315–9, Jul. 2016. https://genesilico.pl/SimRNAweb/

[9] Modeling of Protein-RNA Complex Structures Using Computational Docking Methods.
B. Madan, J. M. Kasprzak, I. Tuszyńska, M. Magnus, K. Szczepaniak, W. K. Dawson, and J. M. Bujnicki Methods Mol. Biol., vol. 1414, no. 21, pp. 353–372, 2016.

[8] RNA 3D Structure Modeling by Combination of Template-Based Method ModeRNA, Template-Free Folding with SimRNA, and Refinement with QRNAS. P. Piatkowski, J. M. Kasprzak, D. Kumar, M. Magnus, G. Chojnowski, and J. M. Bujnicki
Methods Mol. Biol., vol. 1490, no. Suppl, pp. 217–235, 2016.

[7] NPDock: a web server for protein-nucleic acid docking
I. Tuszyńska, M. Magnus, K. Jonak, W. K. Dawson, and J. M. Bujnicki,
Nucleic Acids Research, vol. 43, no. 1, pp. W425–30, Jul. 2015. http://genesilico.pl/NPDock

[6] RNA-Puzzles Round II: assessment of RNA structure prediction programs applied to three large RNA structures
Z. Miao, R. W. Adamiak, M.-F. Blanchet, M. J. Boniecki, J. M. Bujnicki, S.-J. Chen, C. Y. Cheng, G. Chojnowski, F.-C. Chou, P. Cordero, J. A. Cruz, A. R. Ferré-D'Amaré, R. Das, F. Ding, N. V. Dokholyan, S. Dunin-Horkawicz, W. Kladwang, A. Krokhotin, G. Lach, M. Magnus, F. Major, T. H. Mann, B. Masquida, D. Matelska, M. Meyer, A. Peselis, M. Popenda, K. J. Purzycka, A. Serganov, J. Stasiewicz, M. Szachniuk, A. Tandon, S. Tian, J. Wang, Y. Xiao, X. Xu, J. Zhang, P. Zhao, T. Zok, and E. Westhof
RNA, vol. 21, no. 6, pp. 1066–1084, Jun. 2015.

[5] Computational modeling of protein-RNA complex structures
I. Tuszyńska, D. Matelska, M. Magnus, G. Chojnowski, J. M. Kasprzak, L. P. Kozlowski, S. Dunin-Horkawicz, and J. M. Bujnicki
Methods, vol. 65, no. 3, pp. 310–319, Feb. 2014.

[4] Computational modeling of RNA 3D structures, with the aid of experimental restraints
M. Magnus*, D. Matelska*, G. Lach, G. Chojnowski, M. J. Boniecki, E. Purta, W. K. Dawson, S. Dunin-Horkawicz, and J. M. Bujnicki
RNA Biol, vol. 11, no. 5, pp. 522–536, 2014.

[3] MetaLocGramN: A meta-predictor of protein subcellular localization for Gram-negative bacteria
M. Magnus, M. Pawlowski, and J. M. Bujnicki
BBA - Proteins and Proteomics, vol. 1824, no. 12, pp. 1425–1433, Dec. 2012. http://genesilico.pl/MetaLocGramN/

[2] Structural bioinformatics of the human spliceosomal proteome
I. Korneta, M. Magnus, and J. M. Bujnicki,
Nucleic Acids Research, vol. 40, no. 15, pp. 7046–7065, Aug. 2012. http://iimcb.genesilico.pl/SpliProt3D

[1] Structures of MicroRNA Precursors
P. Kozlowski, J. Starega-Roslan, M. Legacz, M. Magnus, and W. J. Krzyzosiak
Current Perspectives in microRNAs (miRNA), no. 1, Dordrecht: Springer, Dordrecht, 2008, pp. 1–16.
</pre>

				</div> <!-- /tab-container -->
   			
   		</div> <!-- twelve -->
   	</div> <!-- row -->   	

   </section> <!-- /info -->




   <!-- footer
   ================================================== -->
   <footer>

   	</div> <!-- /footer-bottom -->

   	<div class="back-to-top">
		 	<a href="#top" class="smoothscroll"><span>Back to Top</span></a>
		</div>

   </footer>

   <div id="preloader"> 
    	<div id="loader"></div>
   </div> 

   <!-- Java Script
   ================================================== --> 
   <script src="js/jquery-2.1.3.min.js"></script>
   <script src="js/plugins.js"></script>
   <script src="js/main.js"></script>

</body>

</html>
